app/google-services.json

```diff
@@ -0,0 +1,55 @@
+{
+  "project_info": {
+    "project_number": "691773829114",
```

Comment:
We really do not want this info public. We need to figure out another way to all have access to this file, without it being open to the public

Reply:
I also asking the same, do we do a git ignore here. I reviewed one.

Reply:
@adevezin I don't know of a way other than switching to a private repo like GitLab.

Reply:
I don't think it will make it to Github using git ignore, but I guess it doesn't have to again once everyone pulled. So delete and then git ignore this from now on?

Reply:
Yeah, it should go to .gitignore :)